CONTRIBUTING.md

```diff
@@ -261,7 +260,52 @@ alias lt='lr test'
 alias lw='lr watch'
 ```
 
-### pkglint
+### Linters
+
+All linters are executed automatically as part of the build script, `yarn build`.
+
+They can also be executed independently of the build script. From the root of a specific package (e.g.
+`packages/@aws-cdk/aws-ec2`), run the following command to execute all the linters on that package -
+
+```bash
+yarn lint
+```
+
+The following linters are used -
+
+- [eslint](#eslint)
+- [pkglint](#pkglint)
+- [awslint](#awslint)
+
+#### eslint
+
+Historically, the CDK has used tslint for linting its typescript source code. With [tslint's deprecation in
+2019](https://medium.com/palantir/tslint-in-2019-1a144c2317a9), we are slowly moving over to using eslint.
+
+It is executed automatically as part of `yarn build`.
+
+To run `eslint` on a specific package, at the root of the package, run
+
+```bash
+eslint . --ext=.ts,.js
+```
+
+All packages in the repo use a standard base configuration found at [eslintrc.js](tools/cdk-build-tools/config/eslintrc.js).
+This can be customized for any package by modifying the `.eslintrc` file found at its root.
+
+If you're using the VS Code and would like to see eslint violations on it, install the [eslint
+extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and add the following to the
+[settings](https://code.visualstudio.com/docs/getstarted/settings).
+
+```json
+"eslint.workingDirectories": [
```

Comment:
That's only needed if you open the entire project in VSCode, right? If you open a single module it should not be required, correct?

Reply:
Correct. I'll clarify this.